From 3a3643d7717570e130d4f70c74992e221912cbba Mon Sep 17 00:00:00 2001
From: Erik Schumacher <erik.schumacher@iris-sensing.com>
Date: Wed, 4 Sep 2024 13:59:22 +0000
Subject: [PATCH] Remove CONFIG_CMD_BOOTEFI for imx8mp_evk_defconfig

As the function "efi_set_bootdev" called within this compiler switch
causes the error msg "**Bad device specification mmc 2#linuxboot_a **".

Signed-off-by: Jan.Hannig <jan.hannig@irisgmbh.de>
---
 configs/imx8mp_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mp_evk_defconfig b/configs/imx8mp_evk_defconfig
index 1827443e51a..27172051a08 100644
--- a/configs/imx8mp_evk_defconfig
+++ b/configs/imx8mp_evk_defconfig
@@ -62,6 +62,7 @@ CONFIG_SYS_CBSIZE=2048
 CONFIG_SYS_PBSIZE=2074
 # CONFIG_BOOTM_NETBSD is not set
 CONFIG_SYS_BOOTM_LEN=0x2000000
+# CONFIG_CMD_BOOTEFI is not set
 # CONFIG_CMD_EXPORTENV is not set
 # CONFIG_CMD_IMPORTENV is not set
 CONFIG_CMD_ERASEENV=y
-- 
2.44.1

